src/OrleansCounterControl/OrleansCounterControl.csproj

```diff
@@ -45,8 +45,24 @@
     <WarningsAsErrors>4014</WarningsAsErrors>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.CodeAnalysis, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
I guess I considered this comment by @jdom a "conclusion" - https://github.com/dotnet/orleans/pull/1115#discussion_r47028940. I don't recall any example of a real issues with the way NuGet puts references to .csproj files resented after that.